src/test/java/nextstep/ladder/LadderTest.java

```diff
@@ -2,15 +2,51 @@
 
 import org.junit.jupiter.api.Test;
 
+import java.util.Map;
+
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
 
 public class LadderTest {
-    public static final int COUNT_OF_PERSON = 5;
-    public static final int HEIGHT = 5;
 
     @Test
-    void newLadder(){
-        assertThat(new Ladder(COUNT_OF_PERSON, HEIGHT).getLines().size()).isEqualTo(HEIGHT);
+    void newLadder() {
+        assertThat(new Ladder(new Players("pobi,honux,crong,jk"), 5, "꽝,5000,꽝,3000").getLines().size()).isEqualTo(5);
+    }
+
+    @Test
+    void play_trueLadder() {
+        Players players = new Players("pobi,honux,crong,jk");
+        String resultString = "꽝,5000,꽝,3000";
+
+        Ladder ladder = new Ladder(players, 5, resultString, () -> true);
+
+        GameResult result = ladder.play(players);
+
+        Map<String, String> results = result.getAllResults();
+
+        assertEquals(players.getCount(), results.size());
```

Comment:
```suggestion
        assertThat(players.getCount()).isEqualTo(results.size());
```
사용하는 기술의 일관성을 생각해보면 `junit`보다는 다른 곳에서 사용하는 `assertj`를 사용하여 일관성을 지켜볼 수도 있을 것 같아요 :)